framework/src/main/java/io/github/kgress/scaffold/webdriver/WebDriverManager.java

```diff
@@ -200,7 +208,8 @@ private MutableCapabilities getDesiredCapabilities() {
                     browserOptions = new EdgeOptions();
                     break;
                 case Opera:
-                    browserOptions = new OperaOptions();
+                    browserOptions = new OperaOptions()
```

Comment:
no option to accept insecure certs here?

Reply:
By default, the `OperaOptions` do not have a method to enable. It is possible to set with `capabilities.setCapability(CapabilityType.ACCEPT_SSL_CERTS, true);`. However, I think the value add is minimal for now.